src/test/regress/expected/multi_create_table_constraints.out

```diff
@@ -6,13 +6,10 @@ ALTER SEQUENCE pg_catalog.pg_dist_jobid_seq RESTART 365000;
 --  test that Citus forbids unique and EXCLUDE constraints on append-partitioned tables.
 CREATE TABLE uniq_cns_append_tables
 (
-	partition_col integer UNIQUE,
+	partition_col integer,
```

Comment:
According to comment above, this(and below) table is used to test that Citus forbids unique and EXCLUDE constraints on append-partitioned tables.

So I think it is better to keep UNIQUE keyword and see error message. It seems that this table is not used anywhere else.